patches/server/0975-Folia-scheduler-and-owned-region-API.patch

```diff
@@ -90,6 +93,7 @@ index 0000000000000000000000000000000000000000..62484ebf4550b05182f693a3180bbac5
 +                throw new IllegalStateException("Already retired");
 +            }
 +            this.tickCount = RETIRED_TICK_COUNT;
++            this.server.entitiesWithScheduledTasks.remove(entity);
```

Comment:
Inappropriate formatting, when accessing field variables you must use the `this` qualifier to clearly indicate the access is for a field